common/lib/dependabot/git_commit_checker.rb

```diff
@@ -270,6 +270,7 @@ def dependency_source_details
         dependency.requirements.map { |r| r.fetch(:source) }.uniq.compact
```

Comment:
Thoughts on selecting only `git` type sources here instead?
```suggestion
        dependency.requirements.map { |r| r.fetch(:source) }.uniq.compact.
        select { |r| source[:type] == "git" }
```